eepromWriter/write_module_memory.py

```diff
@@ -180,37 +190,55 @@ def _parse_model_from_barcode(barcode):
     raise Exception(BAD_BARCODE_MESSAGE.format(barcode))
 
 
+def build_arg_parser():
+    arg_parser = ArgumentParser(
+        description="Firmware & serial uploader for Opentrons modules")
+    arg_parser.add_argument("--write_serial", required=True)
```

Comment:
if you use `action='store_true'` you can later test for `args.write_serial` and it will be `True` if specified and `False` otherwise. Then the user does `--write_serial` (without an argument) when passing it on the commandline.

In addition, if you specify `help='some string'` then when you do `--help` it will print all the arguments with their help strings. Good for discoverability

Reply:
Good call